USERS.md

```diff
@@ -41,84 +41,85 @@ Currently, the following organizations are **officially** using Argo CD:
 1. [GMETRI](https://gmetri.com/)
 1. [Gojek](https://www.gojek.io/)
 1. [Greenpass](https://www.greenpass.com.br/)
-1. [Healy](https://www.healyworld.net)
```

Comment:
I guess some markdown plugin changed the numbering of each line below. Can you please revert unnecessary changes?

Reply:
thank you!